documentation/docs/contributing/ground-rules.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 1
+---
+
+# Ground rules
+
+* For major changes, please open an issue first to discuss what you would like to change. 
+  * Search for existing issues and pull requests on the [project development board](https://github.com/equinor/template-fastapi-react/projects/1) before creating your own.
+* Contributors should fork the repository and work on fixes or enhancements on their own fork.
+  * Use the [pull request feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) to submit your changes to this  repository.
+  * All pull requests should be rebased (with main) and commits squashed prior to the final merge process.
+  * Use [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting for commit messages, so that it's possible to auto-generate the changelogs.
+* Use a separate branch for each issue you’re working on. 
+  * Do not combine fixes for multiple issues into one branch. 
+* Please include unit tests with all your code changes.
+  * All unit tests must be 100% passing before the pull requests will be approved and merged.
```

Comment:
Most of this is included somewhere else, but that might be ok since this is a sort of introduction?